test/requirements.txt

```diff
@@ -13,3 +13,4 @@ protobuf >= 3.9.2, < 3.20
 datasets
 graphviz
 adlfs
+awscli>=1.27.66
```

Comment:
Question: curious why we need to specify the version here?
1. I assume it doesn't work with version prior to `1.27.66`?
2. Does it automatically try to install a version before `1.27.66`?